tmt/cli.py

```diff
@@ -631,12 +631,18 @@ def tests_import(
 @click.option(
     '-d', '--debug', is_flag=True,
     help='Provide as much debugging details as possible.')
+# TODO: move to `template` export plugin options
+@click.option(
+    '--template', metavar='PATH',
+    help="('--how=template' only) Path to a template to use for rendering the export."
```

Comment:
Starting the option help with a parenthesis looks a bit weird. What about adding a note at the end?

```suggestion
    help="Path to a template to use for rendering the export. To be used with '--how=template' only."
```

Reply:
Fixed in 2a807b73